packages/convert-class/index.ts

```diff
@@ -1,50 +1,65 @@
-import compose, { Initializer, PropertyMap, Stamp } from '@stamp/compose';
+import compose from '@stamp/compose';
+
+import type { Initializer, PropertyMap, Stamp } from '@stamp/compose';
 
 const { prototype: functionPrototype } = Function;
 const { assign } = Object;
 const { construct, get, getPrototypeOf, ownKeys, set } = Reflect;
 
-const isClass = (value: unknown): unknown => typeof value === 'function' && /^\s*class\s/.test(value.toString());
+/** @internal */
+interface ObjectWithPrototype extends PropertyMap {
+  prototype: any;
+}
+
+/** @internal */
+const isObjectConstructor = (value: unknown): value is ObjectConstructor =>
+  typeof value === 'function' && /^\s*class\s/.test(value.toString());
 
-const isFunction = (value: unknown): value is Function => value === functionPrototype;
+/** @internal */
```

Comment:
I am unsure what is being tested here and why

Reply:
I don't remember. :) Feel free to rename.